bioconda_utils/docker_utils.py

```diff
@@ -134,6 +120,7 @@
 """
 FROM bioconda/bioconda-utils-build-env
 {self.proxies}
+RUN /opt/conda/bin/conda install -y conda={conda_ver} conda-build={conda_build_ver}
```

Comment:
This is fine for now. I think it would make sense to create a `bioconda-build-setup` (or similar) package at some point which consists of `bioconda_utils-conda_build_config.yaml` and has requirements like
```yaml
requirements:
  run:
    - conda 4.5.4
    - conda-build 3.10.2
    - conda-forge-pinning 2018.05.07
```
and possibly a `post-link.sh` with
```bash
"${CONDA_EXE}" config --env --add channels defaults --add channels conda-forge --add channels bioconda
```
or something.
That way, we could just do a `conda create bioconda-build-setup=version` and don't have to juggle with copying those configuration files manually

Reply:
I don't think so. The nice thing of the current approach is that we can synchronize host and container conda/conda-build without additional facilities. With such a recipe, it will only get out of sync.

Reply:
We could have those perfectly in sync if we install `bioconda-build-setup=version` to either.

(With `conda >=4.4` we could even `conda create bioconda-build-setup=version` and use a completely separate environment for `conda-build` on the host which does not depend on the packages introduced by `bioconda-utils`. I'm not sure about if that approach is that valuable, but at least we could use a smaller Docker image thatn `bioconda-utils-build-env` -- maybe (I have no clue) this could reduce the build time.)

Reply:
Ok, we can keep this in mind, but I still don't see the immediate benefit over what we have here. Seems much simpler to me than another package.